src/scripts/core/visualization-container.js

```diff
@@ -74,6 +74,12 @@
             return this.ctx;
         },
 
+        setVisibility: function (visible) {
+            var node = this.layer.node();
+
+            visible ? node.style.display = 'block' : node.style.display = 'none';
```

Comment:
I **believe** (am fairly certain) it will always be block (since it is an svg element), but I could hunt down some proof if you'd like?